code/style.css

```diff
@@ -1,7 +1,327 @@
+  box-shadow: 3px 3px 20px 5px #DD9296;
+}
+
+/* Flexbox for the form attributes */
+.form { 
+  display: flex;
+  flex-direction: column;
+}
+
+.form-div {
+  margin-bottom: 5px;
+  display: flex;
+  align-items: center;
+}
+
+.form-div label { /* Design of the labels */
+  width: 100px;
+  margin: 5px;
+}
+
+#fullName, #companyName, #mailAdress, #passWord {  /*Styling of the input fields*/
+  width: 250px;
+  border: none;
+  border-bottom: 1px solid black;
+  background-color: #ddeaeb;
+}
+
+.checkbox { /* Flexboxes for the checkboxes */
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.checkbox-child {
+  margin-bottom: 5px;
+}
+
+/** Design of the button **/
+button  { 
+  width: 438px;
+  font-size: 16px;
+  font-weight: bold;
+  background: #ddeaeb;
+  color: #22031F;
+  padding: 8px;
+  margin: 10px;
+  border: 1px solid white; 
+  box-shadow: 2px 2px 5px 2px white; 
+}
+
+button:hover { /** How the button shall look when pointing on it - it gets flat **/ 
+  transform: translateY(5px);
+  box-shadow: none;
+}
+
+/*** ACCORDION ***/
+.accordion {
+  margin: 40px;
+  background-color: #ddeaeb;
+  padding: 10px 15px;
+  display: flex;
+  flex-direction: column;
+  width: 550px;
+  max-height: 690px;
+  border-radius: 5%;
+  cursor: pointer;
+}
+
+.accordion-container-1 {
+  background-color: white;
+  padding-bottom: 20px;
+}
+
+.accordion-container-2 {
+  background-color: #e1e8ed;
+  padding-bottom: 20px;
+}
+
+.accordion:hover { /* When hovering */
+  box-shadow: 3px 3px 20px 5px #DD9296;
+}
+
+.question { /* Design of the question texts */
+  font-weight: bold;
+  line-height: 1em;
+  cursor: pointer;
+  transition: 0.3s;
+  margin: 10px;
+}
+
+.question:hover { /* When hovering the question 'heading */
+  font-size: 15px;
+}
+
+.question::after {
+  content: "\002B";
+  font-size: 24px;
+  float: left;
+  margin-right: 10px;
+}
+
+.question.active::after {
+  content: "\2212";
+}
+
+.question.remove::after {
+  content: "\002B";
+}
+
+.answer-container {
+  height: 0;
+  overflow: hidden;
+  transition: height 0.3s ease; 
+}
+
+.answer-container.active {
+  height: auto;
+}
+
+.answer-text {
+  margin-left: 13px;
+  margin-right: 18px;
+}
+
+footer { /* Footer styling */
+  width: 100%;
+  background-color: #ddeaeb;
+  text-align: center;
+  bottom: 0;
+  color: #22031F;
+  padding: 15px;
+  border-top: 15px solid white;
+}
+
+img { /* Styling of LinkedIn logo */
+  width: 22px;
+  margin-top: 35px;
+  margin-left: 10px;
+}
+
+.footer {
+  font-size: 12px;
+}
+
+/*** Media queries ***/
+@media screen and (max-width: 1121px) { /* Tablet device - I choosed my own measures on width to make the header text fit nicely with the hero-image.*/
+  body {
+    background-color: #d4b4c4;
+  }
+  
+  .hero-image {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    background-image: url("./tabletpic.jpg");
```

Comment:
This was a really nice feature with the responsiveness too, it's super cool to have different themes on different devices!